plugin/src/org/checkerframework/languageserver/eclipse/LSPCheckerFrameworkPreferencePage.java

```diff
@@ -184,6 +293,15 @@ private void storeValues() {
     store.setValue(LSPCheckerFrameworkConstants.CHECKER_PATH, textFieldCheckerPath.getText());
     store.setValue(
         LSPCheckerFrameworkConstants.COMMAND_OPTIONS, textFieldCommandLineOptions.getText());
+    
+    StringBuilder result = new StringBuilder();
```

Comment:
That logic also appears above. Extract into a helper method (that will use StringJoiner).

Reply:
This was just for testing. Just deleted.